### PR TITLE
CEF Display Update

### PR DESCRIPTION
**Pull request by Rosiak**
Got annoyed by the old page, just a new basic page.

#### Please note

> Please read this information carefully.

[X] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)

[X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

Initial Commit

### CODE REVIEW

File: html/pages/routing/cef.inc.php [hunk 1]
```diff
@@ -1 +1,49 @@
...
+            <th>Path</th>
+            <th>Drop</th>
+            <th>Punt</th>
+            <th>Punt2Host</th>
+          </tr>
+        </thead>';
+
+foreach (dbFetchRows('SELECT * FROM `cef_switching` ORDER BY `device_id`, `entPhysicalIndex`, `afi`, `cef_index`') as $cef) {
+    $device = device_by_id_cache($cef['device_id']);
+    $entity = dbFetchRow('SELECT `entPhysicalName`, `entPhysicalModelName`, `entPhysicalContainedIn` FROM `entPhysical` WHERE `device_id` = ? AND `entPhysicalIndex` = ?', array($device['device_id'], $cef['entPhysicalIndex']));
```
Comment (laf) [inline html/pages/routing/cef.inc.php]: You could combine this with the previous query and do a LEFT JOIN

**Comment by laf** [comment]
Definitely all for replacing the crap that was there before - however, I'm personally reluctant to introduce another un-paginated page.

@librenms/reviewers

**Comment by Rosiak** [comment]
@laf I'll see if I can get it done during the week.

**Comment by laf** [comment]
If you can update the sql query I'll merge this but please please submit another PR to paginate the table.

**Comment by f0o** [comment]
:+1: finally getting rid of that useless gif.

I share laf's opinion on non-paginated pages

**Comment by Rosiak** [comment]
No worries, I'll redo it when I have some time

**Pull request merged**